pl_bolts/models/regression/linear_regression.py

```diff
@@ -110,16 +110,17 @@ def add_model_specific_args(parent_parser):
 
 def cli_main():
     from pl_bolts.datamodules.sklearn_datamodule import SklearnDataModule
+    from pl_bolts.utils import _SKLEARN_AVAILABLE
 
     pl.seed_everything(1234)
 
     # create dataset
-    try:
+    if _SKLEARN_AVAILABLE:
         from sklearn.datasets import load_boston
-    except ModuleNotFoundError as err:
-        raise ModuleNotFoundError(  # pragma: no-cover
+    else:  # pragma: no cover
```

Comment:
there an advantage or using "no cover" compare to "no-cover" I was thinking that the joined form is more like a tag :]